gallery/src/components/NxToast/NxToastFullPageExample.tsx

```diff
@@ -0,0 +1,137 @@
+        pharetra convallis posuere morbi leo urna. Tortor posuere ac ut consequat semper viverra nam libero.
+        Volutpat ac tincidunt vitae semper. Pretium fusce id velit ut tortor pretium viverra.
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+        dolore magna aliqua. Tortor consequat id porta nibh venenatis cras. Proin libero nunc consequat interdum
+        varius sit amet mattis. Enim praesent elementum facilisis leo vel. A arcu cursus vitae congue mauris
+        rhoncus aenean vel elit. Morbi tincidunt ornare massa eget egestas. Semper auctor neque vitae tempus quam
+        pellentesque nec nam aliquam. Faucibus nisl tincidunt eget nullam non nisi est sit. Sit amet aliquam id diam
+        maecenas. Porta nibh venenatis cras sed felis eget velit aliquet. Semper feugiat nibh sed pulvinar proin
+        gravida hendrerit lectus a. Auctor neque vitae tempus quam pellentesque nec nam aliquam sem. Purus non enim
+        praesent elementum facilisis leo. Dui sapien eget mi proin. Ultrices eros in cursus turpis massa tincidunt
+        dui ut ornare. Et ligula ullamcorper malesuada proin libero nunc consequat interdum. Turpis egestas maecenas
+        pharetra convallis posuere morbi leo urna. Tortor posuere ac ut consequat semper viverra nam libero.
+        Volutpat ac tincidunt vitae semper. Pretium fusce id velit ut tortor pretium viverra.
+      </NxPageSidebar>
+      <NxPageMain tabIndex={0}>
+        <NxToastContainer>
+          {
+            toasts.map((toast) => (
+              <NxToast key={toast.toastId}
+                       toastId={toast.toastId}
+                       onClose={()=> removeToast(toast.toastId)}>
+                <NxSuccessAlert>Hello World</NxSuccessAlert>
+              </NxToast>
+            ))
+          }
+        </NxToastContainer>
+        <NxPageTitle>
+          <NxH1>Lorem Ipsum</NxH1>
+        </NxPageTitle>
+        <NxTile>
+          <NxTile.Header>Lorem Ipsum</NxTile.Header>
+          <NxTile.Content>
+            <NxP>
+              Loresssm ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore
+              dolore magna aliqua. Tortor consequat id porta nibh venenatis cras. Proin libero nunc consequat interdu
+              varius sit amet mattis. Enim praesent elementum facilisis leo vel. A arcu cursus vitae congue mauris
+              rhoncus aenean vel elit. Morbi tincidunt ornare massa eget egestas. Semper auctor neque vitae tempus qu
+              pellentesque nec nam aliquam. Faucibus nisl tincidunt eget nullam non nisi est sit. Sit amet aliquam id
+              diam maecenas.
+            </NxP>
+            <NxButton type="button" onClick={() => addToast()}>Open Toast</NxButton>
+          </NxTile.Content>
+        </NxTile>
+      </NxPageMain>
+    </>
+  );
+}
```

Comment:
Is there any point in having this example separately from the `NxToastFullPageVariationsExample`?

Reply:
> Is there any point in having this example separately from the `NxToastFullPageVariationsExample`?

We were thinking about showing a "simple" and "variations" example like it was laid out previously, but as you point out, there isn't much point after the refactoring, so I'll remove it.

Reply:
> > Is there any point in having this example separately from the `NxToastFullPageVariationsExample`?
> 
> We were thinking about showing a "simple" and "variations" example like it was laid out previously, but as you point out, there isn't much point after the refactoring, so I'll remove it.

So I ended up just updating the layout of this example to have a "simple layout" example that demonstrates the toast's positioning without NxGlobalHeader presence, as mentioned in your comments: